validate.go

```diff
@@ -202,6 +203,32 @@ func ValidatePatch(p PatchInterface) *field.Error { //nolint: gocyclo // This is
 	return nil
 }
 
+// ValidateCombine validates a Combine.
```

Comment:
You mean we were relying on the code that executes the combine, rather than the validation code? I would guess that's the case. Agreed it's worth back porting.

Reply:
No, I meant on the CRD itself. We were already checking the variables to be `>0` and their FromFieldPath was already not optional, but we were indeed only checking at runtime whether `string` was set and we were not enforcing types to be set explicitly.

Reply:
Oh got it, makes sense.